test/cli/server_test.rb

```diff
@@ -2,10 +2,16 @@
 
 class CliServerTest < CliTestCase
   test "bootstrap" do
-    run_command("bootstrap").tap do |output|
-      assert_match /which curl/, output
-      assert_match /which docker/, output
-      assert_match /apt-get update -y && apt-get install curl docker.io -y/, output
+    SSHKit::Backend::Abstract.any_instance.expects(:execute).with(:docker, :info, raise_on_non_zero_exit: false).returns(true).at_least_once
+
+    assert_equal "", run_command("bootstrap")
+  end
+
+  test "bootstrap with Docker not running" do
+    SSHKit::Backend::Abstract.any_instance.expects(:execute).with(:docker, :info, raise_on_non_zero_exit: false).returns(false).at_least_once
+
+    assert_raise RuntimeError, "Docker is not running on 1.1.1.1, 1.1.1.3, 1.1.1.4, 1.1.1.2. Install it first: https://docs.docker.com/engine/install/" do
+      run_command("bootstrap")
```

Comment:
New bootstrap output indicates where Docker isn't running and how to install it.